### PR TITLE
Adding consistency notes to readme

### PR DESCRIPTION
**Pull request by utka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,22 @@ Example: https://github.com/istio-releases/pipeline/pull/102/files
 And then you can send a PR with this file change to trigger release automation.  As a convention,
 please use `MONTHLY <release version>` as the PR title.
 
+CB_VERIFY_CONSISTENCY makes sure that the same API sha pulled to Proxy and Istio repos.
```
Comment (hklai) [inline README.md]: ...is pulled...

File: README.md [hunk 2]
```diff
@@ -84,6 +84,22 @@ Example: https://github.com/istio-releases/pipeline/pull/102/files
...
+
+To make repos consistent
+Make sure that you are checking for sha in the same branch (ie release-1.1 for
+all repos)
+
+1. In istio/proxy repo in repositories.bzl
+ISTIO_API = abcd // sha from istio/API repo
+ISTIO_API_SHA256 = zxcv
```
Comment (hklai) [inline README.md]: Add instruction for getting sha256.

I think there is an example in istio/proxy

File: README.md [hunk 3]
```diff
@@ -84,6 +84,22 @@ Example: https://github.com/istio-releases/pipeline/pull/102/files
...
+
+2. In istio/proxy repo in istio.deps for ISTIO_API
+lastStableSHA = abcd
+
+3. In istio/istio repo in Gopkg.lock for name = "istio.io/api"
```
Comment (hklai) [inline README.md]: Run deep ensure

File: README.md [hunk 4]
```diff
@@ -84,6 +84,30 @@ Example: https://github.com/istio-releases/pipeline/pull/102/files
...
+wget
+https://github.com/istio/api/archive/COMMIT.tar.gz && sha256sum COMMIT.tar.gz
+
+2. In istio/proxy repo in istio.deps for ISTIO_API
+lastStableSHA = abcd
+
+3. In istio/istio repo in Gopkg.lock for name = "istio.io/api"
+digest = "1:zxcv"
+
+Or for step 3 use [dep](https://github.com/istio/istio/wiki/Vendor-FAQ)
```
Comment (hklai) [inline README.md]: Using dep should be the only recommended way

Comment (utka) [reply README.md]: Done

**Comment by hklai** [comment]
Made an edit directly. PTAL. 
Also please rebase to get the change I made earlier today.

**Comment by utka** [comment]
I think all done.

File: README.md [hunk 5]
```diff
@@ -79,6 +79,29 @@ and you can leave the existing value alone.
...
+[repositories.bzl](https://github.com/istio/proxy/blob/master/repositories.bzl)
+
+ To get sha256 run
+```shell
+wget https://github.com/istio/api/archive/COMMIT.tar.gz && sha256sum COMMIT.tar.gz
+```
+
+ 2. In istio/proxy, update ISTIO_API sha in [istio.deps](https://github.com/istio/proxy/blob/master/istio.deps).
+
+ 3. In istio/istio run [dep](https://github.com/istio/istio/wiki/Vendor-FAQ)
```
Comment (gargnupur) [inline README.md]: Should we point to istio/api location directly here? https://github.com/istio/istio/wiki/Vendor-FAQ#how-do-i-update-istioapi-sha-

Comment (utka) [reply README.md]: I think it is the same thing

**Pull request merged**